addons/mail/static/src/components/call_action_list/call_action_list.xml

```diff
@@ -20,7 +20,7 @@
                                 }"/>
                             </div>
                         </button>
-                        <button class="o_CallActionList_button btn d-flex m-1 border-0 rounded-circle bg-800 shadow-none"
```

Comment:
Isn't it a `bg-dark`?

Reply:
`bg-dark` is darker / lighter in dark mode than `bg-800`. I need to keep this color in both modes, so I removed the class and used the css variable :
https://github.com/odoo-dev/odoo/pull/1327/commits/13840727d418c2d0279fb68f09d36bc778ed07b1#diff-b4cb1840c7645f964aac693ff92c581d8668d3cb9e205049b3ed9eafbf420a59R2